packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -115,7 +116,18 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
 
     this.primaryMobiusUris = [];
     this.backupMobiusUris = [];
+    let mobiusServiceHost = '';
+    try {
+      mobiusServiceHost = new URL(this.webex.internal.services._serviceUrls.mobius).host;
+    } catch (error) {
+      log.warn(`Failed to parse mobius service URL`, {
+        file: CALLING_CLIENT_FILE,
+        method: this.constructor.name,
+      });
+    }
+
     this.mobiusClusters =
+      (mobiusServiceHost && this.webex.internal.services._hostCatalog[mobiusServiceHost]) ||
```

Comment:
Just noticed....Since we are assigning an empty string in mobiusServiceHost, if we do not have 'this.webex.internal.services._serviceUrls.mobius).host' value, mobiusClusters will be empty and won't look for values in catalog. Won't that break the code for other cases

Reply:
Discussed 1:1